_posts/2016-07-03-dart-new-tree-booster-provides-dropout.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "DART: new tree booster provides Dropout"
+date: 2016-07-03 00:00:00 +0900
+author: Yoshinori Nakano
+categories: xgboost
+comments: true
+---
+
+## Introduction
+Gradient Boosting Trees algorithms(especially [XGBoost](https://github.com/dmlc/xgboost)) give awesome performance in lots of [competitions](https://github.com/dmlc/xgboost/tree/master/demo#machine-learning-challenge-winning-solutions).
+They combine a huge number of regression trees with small learning rate.
+
+In this situation, trees added early are significance and trees added late are unimportant.
+DART algorithm drops trees added earlier to level contributions.
+
+This post announces that DART has been now implemented in XGBoost.
+
+## Original paper
+Rashmi Korlakai Vinayak, Ran Gilad-Bachrach. "DART: Dropouts meet Multiple Additive Regression Trees."
+
+[JMLR](http://www.jmlr.org/proceedings/papers/v38/korlakaivinayak15.pdf)
+
+## Features
+- Drop trees in order to solve the over-fitting.
+  - Trivial trees (to correct trivial errors) may be prevented.
+- Slower than `gbtree`.
```

Comment:
Note that due to the randomness in dropout. Currently dart runs slower than gbtree, and cannot directly benefit from early stopping(because evaluation metric fluctuates).